CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ and the versioning aims to respect [Semantic Versioning](http://semver.org/spec/
 - nuclear, nuclear electric hydrogen, nuclear electrical energy, solar electric hydrogen, solar electrical energy (#1559)
 - electrical energy share, renewable energy share value, electrical energy share value (#1561)
 - has energy main/auxilary input, has energy main/waste output (#1564)
+- life cycle assessment (#1576)
```

Comment:
```suggestion
```

Reply:
Here is the section of the last release. The changes now should be part of the unpublished section above, line 10.